estuary/api/v1.py

```diff
@@ -276,7 +276,19 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    if item.id_:
```

Comment:
Checking this will throw an AttributeError. I would suggest using something like

 if item.__label__ == DistGitCommit.__label__:
    /* use item.hash_
else:
   /* use item.id_